scripts/ci/docker-compose/integration-presto.yml

```diff
@@ -18,9 +18,30 @@
 version: "2.2"
 services:
   presto:
-    image: prestosql/presto:330
+    build:
+      context: ..
+      dockerfile: ./docker-files/presto/Dockerfile
```

Comment:
Love it - this then will build as needed ! We should do the same with other images we use for Devel.